CONTRUBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 Development is on [GitHub](https://github.com/cilium/ebpf) and contributions in
 the form of pull requests and issues reporting bugs or suggesting new features
-are welcome.
+are welcome. Take a look at [the architecture](ARCHITECTURE.md) of the library.
```

Comment:
Maybe reword to "Please take a look at the architecture of the library for high-level guidance on the development goals of the library."